chutney/engine/src/test/java/com/chutneytesting/engine/domain/execution/strategies/RetryWithTimeOutStrategyTest.java

```diff
@@ -317,4 +316,79 @@ private void stopExecution(ScenarioExecution scenarioExecution) {
         ReflectionUtils.setField(stopField, scenarioExecution, true);
     }
 
+    @Test
+    public void should_evaluate_spel_expression_in_timeout() {
```

Comment:
Please do only one test with a @ParameterizedTest

Reply:
I would add that the assertion on info is out of the thing to test here i think. The feature has to test that the stepEvaluator of the step is used, and only that. Just let the test be in error if the `when` clause is not called. And if you **really** want to have an assertion, just verify this call then.